src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseExclusions.txt

```diff
@@ -13,8 +13,13 @@
 #
 
 # False positive
+src/arcade/Documentation/UnifiedBuild/Foundational-Concepts.md
 src/arcade/src/Microsoft.DotNet.Build.Tasks.Installers/src/BuildFPMToolPreReqs.cs|json
 src/arcade/src/Microsoft.DotNet.Build.Tasks.Installers/build/rpm_templates/copyright|cecill-c
+src/arcade/src/SignCheck/SignCheck/THIRD-PARTY-NOTICES.TXT
```

Comment:
I am trying out a quick fix to see if we can delete this file: https://github.com/dotnet/arcade/pull/14090